public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx

```diff
@@ -209,6 +211,7 @@ export const QueryWrapper = ({
           onAddQuery={() => onDuplicateQuery(cloneDeep(query))}
           onRunQuery={onRunQueries}
           queries={editorQueries}
+          range={range}
```

Comment:
is it possible to avoid this prop drilling and use this range property only in the QueryEditorRow: we have the app to figure out if we want to use the `getTimeSrv().timeRange()` or the `rangeUtil.relativeToTimeRange(query.relativeTimeRange ?? getDefaultRelativeTimeRange());`

Reply:
I'd refrain from doing that. This way we would move more alerting-specific logic from our own components to core query components. Our logic should stick in our code and components as long as possible

Reply:
ahh, good point! it makes sense, then.